### ISSUE TITLE
Powershell CBH indentation issue

### ISSUE THREAD

**Issue by RobDesideri** [stkb/Rewrap#43]
Hi,
If i wrap an entire powershell script containing the comment-based-help section, all LF are removed.

# Example

## Input

```
<#
.SYNOPSIS
  Return a configuration object from a yml file.
.DESCRIPTION
  Get all configuration settings from a yml file, rewrite relative path to absolute path, return an
  hashtable object.
.PARAMETER YamlPath
  Optional - Path of the yaml configuration file.
.EXAMPLE
  $config = .\GetConfig
  $config is filled by configuration settings from ProjectRoot\config\project.yml file.
.EXAMPLE
  $config = .\GetConfig .\myConfig.yml
  $config is filled by configuration settings from .\myConfig.yml file.
.INPUTS
  None.
.OUTPUTS
  Hashtable object containing all configuration settings.
#>
```

## Output (after Rewrap action)

```
<#
.SYNOPSIS Return a configuration object from a yml file. .DESCRIPTION Get all configuration settings
  from a yml file, rewrite relative path to absolute path, return an hashtable object. .PARAMETER
  YamlPath Optional - Path of the yaml configuration file. .EXAMPLE $config = .\GetConfig $config is
  filled by configuration settings from ProjectRoot\config\project.yml file. .EXAMPLE $config =
  .\GetConfig .\myConfig.yml $config is filled by configuration settings from .\myConfig.yml file.
  .INPUTS None. .OUTPUTS Hashtable object containing all configuration settings.
#>
```

## Expected output

```
<#
.SYNOPSIS
  Return a configuration object from a yml file.
.DESCRIPTION
  Get all configuration settings from a yml file,
  rewrite relative path to absolute path, return
  an hashtable object.
.PARAMETER YamlPath
  Optional - Path of the yaml configuration file.
.EXAMPLE
  $config = .\GetConfig
  $config is filled by configuration settings from
  ProjectRoot\config\project.yml file.
.EXAMPLE
  $config = .\GetConfig .\myConfig.yml
  $config is filled by configuration settings from
  .\myConfig.yml file.
.INPUTS
  None.
.OUTPUTS
  Hashtable object containing all configuration settings.
#>
```

**Comment by stkb** [comment]
Ah, that isn't supported yet. I will get onto it.

**Comment by stkb** [comment]
Hi this is now in [v1.4.2](https://github.com/stkb/Rewrap/releases/tag/v1.4.2). I tested it on your example, and more details are also available [here](https://github.com/stkb/Rewrap/blob/857c89bf43555e8ccb804a7ac289b4f2d38ab35e/specs/Basics/3.4-PowerShell-CBH.md).

Let me know if you still have any issues!

**Comment by RobDesideri** [comment]
Hi @stkb, good job!!

Just one consideration:
The "_NOTES_" field is often used to add additional details, such as author name, version, date of creation, etc.

Here one examples of this use: [PowerShell Script Template](https://gist.github.com/9to5IT/9620683).

At now, if I rewrap the CBH, all EOLs in the _NOTES_ field are removed, so this:

```
.NOTES
  Author:         Roberto Desideri
  Contacts:       http:\\robdesideri.it
  Creation Date:  July 31st 2017, 3:06:54 am
```

Becomes:

```
  Author:         Roberto Desideri Contacts:       http:\\robdesideri.it Creation Date:  July 31st
  2017, 3:06:54 am
```

What do you think about it?

**Comment by stkb** [comment]
Yeah I saw that form in a couple of the scripts that I looked at, but didn't know if it was a standard thing. This is a tricky case and I need to think a bit about it.

In the mean time there are a few workarounds possible:
1. Put two spaces at the end of each of those lines. This will make sure the newlines after them are kept.
2. Put a `* + -` or `#` in front of each line.
3. Only select the lines in the comment you want to be wrapped before wrapping.
4. [Set Rewrap's `wholeComment` setting to `false`](https://github.com/stkb/Rewrap/wiki/Settings-VSCode#wrapping-whole-or-parts-of-comments). Then only the current paragraph the cursor is in, will be wrapped, and not the whole comment.
